### DISCUSSION TITLE
Get mutual friends

### DISCUSSION THREAD

**Discussion by EduardSchwarzkopf** [kevinzg/facebook-scraper#397] [Q&A] [answered]
Hey sry again for asking, but could you tell me how i'm able to get mutual friends of 2 users?
facebook has this url where you can check, but i couldn't find anything similar in the code:
https://www.facebook.com/browse/mutual_friends/?uid=<user_id1>&node=<user_id2>

Thanks a ton!

**Comment by neon-ninja** [comment, marked as answer]
This would only show you mutual friends for the account you're using. As most people use dummy accounts with 0 friends, this wouldn't make sense for the scraper.

**Reply by EduardSchwarzkopf** [reply]
i've just tested this with one random account: Picked his id and one of his family members and i could see the mutual friends. No connection between my fake account and his account. Could you please confirm that?

**Reply by neon-ninja** [reply]
Can you share an actual working link?

**Reply by EduardSchwarzkopf** [reply]
sure, here is a link like described:
https://www.facebook.com/browse/mutual_friends/?uid=100000329819079&node=100000000501045

**Reply by EduardSchwarzkopf** [reply]
as you can see, you are even able to see the mutual friends, even though the second user has disabled the friendslist
